ix-dev/community/windmill/templates/Caddyfile

```diff
@@ -0,0 +1,18 @@
+{
+	layer4 {
+		:{{ values.network.smtp_port }} {
+			proxy {
+				to {{ values.consts.windmill_server_container_name }}:2525
+			}
+		}
+	}
+}
+
+{$BASE_URL} {
```

Comment:
why not directly set this via jinja instead of variables like the rest of the file?

Reply:
moved to jinja